tensorboard/plugin_util.py

```diff
@@ -201,7 +201,11 @@ def proto_to_json(proto):
     Args:
       proto: The proto to convert to JSON.
     """
-    current_version = metadata.version("protobuf")
+    # Fallback for internal usage.
+    try:
+        current_version = metadata.version("protobuf")
+    except metadata.PackageNotFoundError:
+        current_version = "5.0.0"
```

Comment:
The "current version" in this case would actually be whatever version we have internally, right? (which should be close to the newest version available, eventually larger than this 5.0.0).

Let's add a comment to clarify that this fall-back version number would likely be inaccurate, but it's safe to assume it would be at least this version internally, which we care to use the appropriate parameter below.

Reply:
We don't really have "versions" for non-3p code in the monorepo, so this is just choosing a version number that will use the non-deprecated kwarg internally. Added a comment clarifying as such.